incubator/mongodb/README.md

```diff
@@ -0,0 +1,192 @@
+# MongoDB Helm Chart
+
+## Prerequisites Details
+* Kubernetes 1.3 with alpha APIs enable
+* PV support on the underlying infrastructure
+
+## PetSet Details
+* http://kubernetes.io/docs/user-guide/petset/
+
+## PetSet Caveats
+* http://kubernetes.io/docs/user-guide/petset/#alpha-limitations
+
+# TODO
+* Set up authorization between replicaset peers.
+* Set up sharding.
+
+## Chart Details
+
+This chart implements a dynamically scalable mongoDB replicaset 
```

Comment:
This is how MongoDB refers to an HA cluster (i.e. multi-node).. so best to clarify this and note that it's not a K8s ReplicaSet ;). It's a PetSet of MongoDB Pods.